test/e2e_node/system/kernel_validator_helper.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package system
+
+// KernelValidatorHelper is an interface intended to help with os specific kernel validation
+type KernelValidatorHelper interface {
+	// GetKernelRelease gets the current kernel release version of the system
+	GetKernelRelease() ([]byte, error)
```

Comment:
`GetKernelVersion` or `GetKernelReleaseVersion` is more precise.

Also, I think the function should just return a string. Any trimming and byte array conversation should be done in the function.